### DISCUSSION TITLE
Overleaf keyboard shortcuts

### DISCUSSION THREAD

**Discussion by xteeven** [overleaf-workshop/Overleaf-Workshop#129] [Q&A]
Hi All, I've been using the extension, and it's fantastic. 

Now, when I work in Overleaf, I use the shortcuts to make the text bold or italics, for example; however, in the VScode environment, I cannot find these shortcuts or redefine them. 

Is this feature implemented?

**Comment by iamhyc** [comment]
Hi, we don't have those Overleaf shortcuts implemented. I also think it is not necessary to insert `\textit` or `\textbf` via shortcuts.

**Comment by ustcwl** [comment]
> Hi All, I've been using the extension, and it's fantastic.
> 
> Now, when I work in Overleaf, I use the shortcuts to make the text bold or italics, for example; however, in the VScode environment, I cannot find these shortcuts or redefine them.
> 
> Is this feature implemented?

I think you can use this  https://github.com/OrangeX4/hsnips
